.github/workflows/hosted_runners.yml

```diff
@@ -1196,7 +1217,12 @@ jobs:
       working-directory: ${{ steps.build_paths.outputs.BINARY }}
       shell: cmd
       run: |
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars${{ matrix.bitness }}.bat"
+        if ${{ matrix.bitness }}==arm64 (
+           echo Arm64 binaries can't be run on x64
+           exit 0
+        )
```

Comment:
nit: (Can clean up later if you'd like) lets just use the Github Action `if` as for the MSI case.

Reply:
Will clean up this in this PR